appveyor.yml

```diff
@@ -0,0 +1,45 @@
+environment:
+  password:
+    secure: uOcv5TOm/CgwOmiLQhNX+g==
```

Comment:
If you want to appveyor to deploy the wheel to pypi, you'll need to create an encrypted variable for "password=the_zope.wheelbuilder_password" and let me know what it is, I'll incorporate it into the script. https://www.appveyor.com/docs/build-configuration#secure-variables